### ISSUE TITLE
Viewing events across monitors stops when 2 events start at the same time.

### ISSUE THREAD

**Issue by poldon** [ZoneMinder/zoneminder#2195]
When I click an event number (i.e. the total number of events for the past hour on all monitors/cameras) and then click the first event to go to the player page, the Next and/or Previous button will be disabled when I hit an event that started at the same time as another event on a different camera.

Here's an example set of events (Note the event numbers are out of order when they start at the same time):
![event list](https://user-images.githubusercontent.com/12818275/44880613-9ae8d700-ac72-11e8-8a60-9c962f83109d.png)

If I try to play the first one, the Next button is disabled:
![broken player](https://user-images.githubusercontent.com/12818275/44880671-c8ce1b80-ac72-11e8-8456-fa19e147b326.png)

This makes reviewing a long sequence of events very time consuming.  It took me a while to track down the issue.  Hopefully, it's an easy fix.

**ZoneMinder Version (`zmaudit.pl -v`):** 1.31.47

**Are you using a development snapshot / git checkout?  If so, what is the latest commit? (`git rev-parse HEAD`):** package from zoneminder-master PPA

**Linux Distribution and Version (`cat /etc/os-release` or `cat /etc/redhat-release`):** Ubuntu 18.04.1 LTS

**If the issue concerns a camera, provide the make, model, frame rate, resolution and ZoneMinder Source Type:** N/A

**Relevant log lines:**
```
None
```

**Comment by connortechnology** [comment]
Can you confirm that this is still an issue? I have been unable to make it happen and I have recently made a change to how events jump from one to another.

**Comment by poldon** [comment]
Yes, it's still an issue.  I just updated to 1.31.47~20180909174905-bionic.

**Comment by poldon** [comment]
Have there been any updates on this?  It might be a while before I can try and troubleshoot it myself.

**Comment by connortechnology** [comment]
Please note that the latest version is 1.32.0.  

My system handles this case perfectly.  I have push a small fix to sort by Id as well as time, so now the Id's should be in the correct order.

**Comment by connortechnology** [comment]
Apparently I am not as smart as I think I am.  Building new packages now which should fix it.
